.cargo/config

```diff
@@ -1,5 +1,29 @@
 [alias]
 xtask = "run --package xtask --"
+ruma-clippy = """\
+  clippy -p ruma --all-targets --all-features --quiet -- \
+    -W missing_copy_implementations \
```

Comment:
I've removed a lot of Copy implementations intentionally so we are less likely to be forced into breaking changes to support new things. I think this lint should go (and the related changes).

Also, why are some lints here using underscores and some dashes?